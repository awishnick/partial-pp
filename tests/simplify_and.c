// RUN: -DDEFINED -DYES=1 -UUNDEFINED -DNO=0
#if YES && defined(DEFINED)
// yes
#endif

#if defined(DEFINED) && YES
// yes
#endif

#if defined(UNDEFINED) && YES
// no
#endif

#if YES && NO
// no
#endif

#if defined(NO) && YES
// yes
#endif
