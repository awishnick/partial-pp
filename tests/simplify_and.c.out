// RUN: -DDEFINED -DYES=1 -UUNDEFINED -DNO=0
// yes

// yes



// yes
